src/textual/app.py

```diff
@@ -561,7 +561,7 @@ def validate_sub_title(self, sub_title: Any) -> str:
 
     @property
     def workers(self) -> WorkerManager:
-        """The [worker](guide/workers/) manager.
+        """The [worker](/guide/workers/) manager.
```

Comment:
I don't think absolute links are well supported. It might work locally but break once deployed. Unfortunately MkDocs link validation features won't work in docstrings, but in a Markdown page (and therefore in the docstring too) you'd ideally use `[worker](guide/workers.md)` or similar.